regression-tests.dnsdist/test_Advanced.py

```diff
@@ -1279,6 +1279,42 @@ def testRefusedViaLua(self):
         refusedResponse.id = receivedResponse.id
         self.assertEquals(receivedResponse, refusedResponse)
 
+class TestAdvancedLuaActionReturnSyntax(DNSDistTest):
+
+    _config_template = """
+    function refuse(dq)
+        return DNSAction.Refused
+    end
+    addAction(AllRule(), LuaAction(refuse))
+    newServer{address="127.0.0.1:%s"}
+    """
+
+    def testRefusedWithEmptyRule(self):
+        """
+        Advanced: Short syntax for LuaAction return values
+        """
+        name = 'refused.advanced.tests.powerdns.com.'
```

Comment:
Would you mind using a different qname than the previous test? It often makes debugging easier.